src/Microsoft.AspNet.Html.Abstractions/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+    "version": "1.0.0-*",
+    "description": "ASP.NET 5 HTML content interface.",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/aspnet/httpabstractions"
+    },
+    "dependencies": {
+        "Microsoft.Framework.WebEncoders.Core": "1.0.0-*"
```

Comment:
You mean `WebUtilities` package? I could use that, but this has all the essential interfaces. And I found the `Core` package to be used in other Abstractions projects as well..